docker/localnode/scripts/step5_start_sei.sh

```diff
@@ -8,17 +8,5 @@ mkdir -p $LOG_DIR
 # Starting sei chain
 echo "Starting the sei chain daemon"
 cp build/generated/genesis-sei.json ~/.sei/config/genesis.json
-./build/seid start --chain-id sei > "$LOG_DIR"/seid-$NODE_ID.log 2>&1 &
+seid start --chain-id sei > "$LOG_DIR/seid-$NODE_ID.log" 2>&1 &
 echo "Done" >> build/generated/launch.complete
-
-echo "SeiNode $NODE_ID started successfully! Check your logs under $LOG_DIR/"
-
-until [ $(cat build/generated/launch.complete |wc -l) = 4 ]
```

Comment:
why do we remove this?

Reply:
This is moved to the deploy.sh script:https://github.com/sei-protocol/sei-chain/pull/514/files/7a5cc77fd8453acee9d27ec0985e8e7af46272b0#diff-d22eb0ea479ed5fe9f57902675dbfc58979fc4abe2ff50b2ee1f382d12349810R63